src/keybindings/defaults.rs

```diff
@@ -4,98 +4,100 @@ use winit::event::{ModifiersState, VirtualKeyCode as VirtKey};
 
 const IS_MACOS: bool = cfg!(target_os = "macos");
 
-pub fn defaults() -> Keybindings {
-    let ctrl_or_command = if IS_MACOS {
-        ModifiersState::LOGO
-    } else {
-        ModifiersState::CTRL
-    };
+impl Default for Keybindings {
+    fn default() -> Self {
+        let ctrl_or_command = if IS_MACOS {
+            ModifiersState::LOGO
+        } else {
+            ModifiersState::CTRL
+        };
```

Comment:
I think I'd prefer something like `pub fn defaults() -> Vec<(Action, KeyCombo)>` in this file still and then

```rust
impl Default for Keybindings {
    fn default() -> Self {
        Self(defaults::defaults())
    }
}
```

in mod.rs just to try and keep impls contained in the same file

I definitely like making `Keybindings` into a proper type though!